db/column_family.cc

```diff
@@ -616,7 +619,7 @@ void ColumnFamilyData::RecalculateWriteStallConditions(
                imm()->NumNotFlushed() >=
                    mutable_cf_options.max_write_buffer_number - 1) {
       write_controller_token_ =
-          SetupDelay(ioptions_.delayed_write_rate, write_controller,
+          SetupDelay(mutable_db_options_.delayed_write_rate, write_controller,
```

Comment:
Instead of storing mutable_db_options in ColumnFamilyData, I think there are two options:

1. store mutable_db_options.delayed_write_rate as max_write_rate in write_controller on initialization and update it if SetDBOptions() is called. Then we use the value stored in write_controller when needed.

2. Alternatively, we can pass mutable_db_options from DBImpl all the way to ColumnFamilyData::RecalculateWriteStallConditions() when the method is being called.

I think 1 would look cleaner than option 2 though.